### ISSUE TITLE
Recent file is not opening and images are not loading

### ISSUE THREAD

**Issue by matthieumartin2812** [chocoford/ExcalidrawZ#54]
Hello,

First of all, thanks a lot for this amazing application.

Recently i faced 2 unexpected bugs with your application.

The data of this file is not loaded anymore, see the screenshot below
<img width="560" alt="image" src="https://github.com/user-attachments/assets/3019d6b3-49a3-4759-91ff-766a69055ed0">

For information, I didn't move the file origin path, and i don't understand why this file is not loaded anymore.
It is quite annoying because i lost this useful 

Besides that, i worked last friday and a new excalidraw file and this morning the images linked to it are not loading (see screenshot below)
<img width="577" alt="image" src="https://github.com/user-attachments/assets/cb877367-340c-4790-aa86-2d7913a1c4bf">

However, it seems that the media file is into the app because i see it into the settings->media files.
Do you have a solution for me please ?

Thanks a lot for your reply.

**Comment by chocoford** [comment]
Hello, @matthieumartin2812. I’m sorry to hear that your experience has been disrupted.

This seems to be an issue with data format compatibility. If possible, could you provide me with the problematic file? Alternatively, you could share a similar file that has the same issue, if the file in question contains sensitive information.

From what I can see, your data should still be safe. If you have closed the app at any point, a backup of the last session should be available in “Settings - Backup.”

**Comment by matthieumartin2812** [comment]
Into the backups settings directory, i have only found 2 files (only this 2 days)
![image](https://github.com/user-attachments/assets/64529b06-a6ed-4c62-99c0-a33d99fe645a)

Are this backups are generated when we quit the application ?

I'm sorry i dont know where the saved file is stored by your application...

**Comment by chocoford** [comment]
Yes, to ensure the safety of your data, the app employs multiple safeguards (including File History and Backup).

However, the issue you encountered still needs to be addressed urgently. If you have the time, you could export the problematic file and send it to me. This would greatly help in identifying and resolving the issue.

**Comment by matthieumartin2812** [comment]
Here is the excalidraw file where the images can't load into it.
The other file was not saved...
[ExcalidrawZ exported at 2 Dec 2024 at 17:08.zip](https://github.com/user-attachments/files/17980709/ExcalidrawZ.exported.at.2.Dec.2024.at.17.08.zip)

**Comment by chocoford** [comment]
Does “The other file was not saved…” mean it wasn’t backed up? If so, do these files still appear in your current sidebar?

The excalidraw file is fine when it is exported.

By the way, which version of ExcalidrawZ are you currently using?

**Comment by matthieumartin2812** [comment]
> Does “The other file was not saved…” mean it wasn’t backed up? Do these files still appear in your current sidebar?
Yeah that's it, it was not saved and is not in the backups folder

> The excalidraw file is fine when it is exported.
> 
> By the way, which version of ExcalidrawZ are you currently using?
I am using the version 1.2.4 (27)
